tests/test_omi_calculator.py

```diff
@@ -542,9 +643,19 @@ def test_initiate_eof_post_processing():
     eofs = omi.initiate_eof_post_processing(raw_eofs, eofs_postprocessing_type="kiladis2014",
                                             eofs_postprocessing_params=None)
 
-    # Check EOFRotation-PP
-    # ToDo: (Sarah): Add a similar test for your pp here to make sure, that it is actually executed by the initiation routine...
-    # ... I am aware of the fact that this tests the numbers as the specific unit tests in the sepearate file for your pp routine...
-    # ... However, I thin it it important to test the the right pp routine is actually executed by the initiation routine.
+    # Check rotation pp-routine
+    rotation_params = {"sign_doy1reference": True}
+
+    eofs = omi.initiate_eof_post_processing(raw_eofs, eofs_postprocessing_type="eof_rotation",
+                                            eofs_postprocessing_params=rotation_params)
+
+    # Validate EOFs against rotated reference (results should be equal)
+    mjoindices_reference_eofs_rotated = eof.restore_all_eofs_from_npzfile(mjoindices_reference_eofs_rotated_filename)
+    for idx, target_eof in enumerate(eofs.eof_list):
+        if not mjoindices_reference_eofs_rotated.eof_list[idx].close(target_eof):
+            errors.append("EOF rotation PP Test: EOF data at index %i is incorrect" % idx)
 
+    # Test that default settings work for rotation pp-routine
+    eofs = omi.initiate_eof_post_processing(raw_eofs, eofs_postprocessing_type="eof_rotation")
+    
```

Comment:
Is it correct that the eofs object created with the last line is not checked anymore?

Reply:
added check of eof object